src/ucp/core/ucp_context.c

```diff
@@ -243,6 +243,11 @@ static ucs_config_field_t ucp_config_table[] = {
    "Maximal number of devices on which a rendezvous operation may be executed in parallel",
    ucs_offsetof(ucp_config_t, ctx.max_rndv_lanes), UCS_CONFIG_TYPE_UINT},
 
+  {"MIN_RNDV_CHUNK_SIZE", "16kb",
+   "Minimal chunk size to split the message sent with rendezvous protocol on\n"
```

Comment:
I'd prefer to use `minimum` instead `minimal`. I know that there are many `minimal` in the documentation and comments in the code. However, once I got the following comment https://github.com/openucx/ucx/pull/6557#discussion_r604469311, I use `maximum` instead `maximal`. ( But I'm not 100% sure that minimum/minimal is the same case).